prisma/schema/application.prisma

```diff
@@ -19,29 +14,52 @@ enum NextAction {
   CANDIDATE_INFORMED
 }
 
+enum QualificationType16 {
+  GCSE
+  INDIAN_X
+  MALAYSIAN
+  SINGAPORE
+  OTHER
+}
+
+enum QualificationType18 {
+  A_LEVEL
+  INDIAN_XII
+  MALAYSIAN
+  SINGAPORE
+  ROMANIAN
+  OTHER
+}
+
 model Application {
-  id                    Int             @id @default(autoincrement())
-  applicant             Applicant       @relation(fields: [applicantId], references: [id])
+  id                    Int                  @id @default(autoincrement())
+  applicant             Applicant            @relation(fields: [applicantId], references: [id])
   applicantId           Int
   admissionsCycle       Int
   applicationDate       DateTime
   wideningParticipation Boolean
   hasDisability         Boolean
   feeStatus             FeeStatus
-  tmuaPaper1Score       Decimal         @db.Decimal(2, 1)
-  tmuaPaper2Score       Decimal         @db.Decimal(2, 1)
-  tmuaOverallScore      Decimal         @db.Decimal(2, 1)
-  qualifications        Qualification[]
-  nextAction            NextAction      @default(TMUA_CANDIDATE)
-  reviewer              User?           @relation(fields: [reviewerId], references: [id])
+  tmuaPaper1Score       Decimal?             @db.Decimal(2, 1)
+  tmuaPaper2Score       Decimal?             @db.Decimal(2, 1)
+  tmuaOverallScore      Decimal?             @db.Decimal(2, 1)
+  age16ExamType         QualificationType16?
+  age16Score            Decimal?             @db.Decimal(3, 1)
+  age18ExamType         QualificationType18?
+  age18Score            Decimal?             @db.Decimal(3, 1)
+  imperialReview        ImperialReview?
+  nextAction            NextAction           @default(TMUA_CANDIDATE)
+  reviewer              User?                @relation(fields: [reviewerId], references: [id])
   reviewerId            Int?
 }
 
-model Qualification {
-  id            Int               @id @default(autoincrement())
-  type          QualificationType
-  title         String
-  score         Decimal           @db.Decimal(3, 1)
-  application   Application       @relation(fields: [applicationId], references: [id])
-  applicationId Int
+model ImperialReview {
```

Comment:
Remind me again: according to our investigation, what did this table represent exactly?
I am asking so as to come up with a more self-explanatory name for the table and its attributes.

Reply:
The `ImperialReview` model was meant to represent scores, comments and other feedback given by a reviewer (as opposed to TMUA results for example which are external).

Reply:
I would then suggest:

- `InternalReview` for the model
- `motivationAdminScore` (and then `motivationReviewerScore`, eventually) 
- `extracurricularAdminScore` (and then `extracurricularReviewerScore`, eventually) 
- `lastEditedBy` and `lastEditedOn` instead of `examsRating*`. Whether these two in particular are needed we should ask Ritu. If they want better traceability, configuring suitable logging could be best.

I would also recommend checking this library out to sync table and attribute comments with Postgres comments: https://github.com/onozaty/prisma-db-comments-generator

I think that given the context that is generally needed to make sense of these tables and attributes, this could be well worth using, starting from this very table in this PR.

Reply:
names addressed in e92e52aeb90ede3c67559c60d721fa3f9e3be805